adding details on setting default auth providers and idps (#776)
chainguard-dev/edu: content/chainguard/chainguard-enforce/authentication/custom-idps/index.md

chainctl auth login --identity-provider=$IDP_ID
```

You can retrieve all your identity provider's unique IDs by running `chainctl iam identity-providers list`.

> Remembering identity provider IDs like this can be difficult. As an alternative, you can set the identity provider by editing the `chainctl` configuration file. You can do so with the following command.

Reviewer: What about...

```suggestion
As an alternative to remembering identity provider IDs, you can set the identity provider by editing the `chainctl` configuration file. You can do so with the following command.
```

chainctl iam identity-provider list
```

> This will return a list of details for each of your identity providers, including their names and unique IDs, 

Reviewer: ```suggestion
This will return a list of details for each of your identity providers, including their names and unique IDs.
```
